packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -84,6 +86,34 @@ class ExpansionPanel {
   ///
   /// Defaults to false.
   final bool isExpanded;
+
+}
+
+///An extension of the expansion panel class that allows for radio functionality.
+///
+///This type of panel will automatically handle closing and opening itself, so
+///setting the [isExpanded] value on these panels will have no effect. A unique
```

Comment:
They can't set this value though since it is not present in the constructor.  Maybe just "isExpanded" cannot be set, instead use [initiallyExpanded] to ....